frontend/index.html

```diff
@@ -1,16 +1,22 @@
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
This site should work without internet access, so this should be vendored or some other solution should be found.
What are these used for?

Reply:
> Looks great so far, some notes:
> 
>     1. We are using yarn, so please remove the `package-lock.json`, also please remove any files not needed, like the `yarn-error.log`
> Required refinements done for this
>     2. If one selects more than the available rows for the number of displayed rows, the table adds unnecessary padding to the bottom.
> Extra padding removed from material table
>     3. The camera load should probably go into the `System Information` tab
> Done 
>     4. Please graph the historic change of the load and not just the current value. Sorry if the description was unclear there.
> Got your point replaced bar chart with a multi line chart.
>     5. What is the reason for omitting the `BSSID`?
Not getting that as output , I did'nt ommit intentionally.